src/EA.Weee.Web/ViewModels/Shared/ViewTransferNoteViewModel.cs

```diff
@@ -43,5 +43,9 @@ public override string TabName
         }
 
         public bool ReturnToView { get; set; }
+
+        public string ApprovedTransfererDetails { get; set; }
```

Comment:
Remove, just use the existing properties and and set the address fields the same way we did for the the evidence view model.

for example 
![image](https://user-images.githubusercontent.com/43237115/193301742-696458c7-206f-48b4-b211-a00fdcd3c707.png)